packages/beacon-node/src/chain/stateCache/stateContextCache.ts

```diff
@@ -84,6 +84,29 @@ export class StateContextCache {
     return this.cache.size;
   }
 
+  /**
+   * 
+   */
+  updateUnfinalizedPubkeys(validators: UnfinalizedPubkeyIndexMap, epoch: Epoch): void {
+    const cpKeySets = Array.from(this.epochIndex.entries())
```

Comment:
no need to pass in an epoch as this cache only stores unfinalized states anyway, see `deleteAllBeforeEpoch`